interfaces/builtin/microstack_support.go

```diff
@@ -102,14 +102,23 @@ const microStackSupportConnectedPlugAppArmor = `
 # Used by libvirt (cgroup-related):
 /sys/fs/cgroup/unified/cgroup.controllers r,
 /sys/fs/cgroup/cpuset/cpuset.cpus r,
+# cgroups v2 used by libvirt and nova-compute:
+/sys/fs/cgroup/cgroup.controllers r,
+/sys/fs/cgroup/cgroup.subtree_control rw,
 
 # Non-systemd layout: https://libvirt.org/cgroups.html#currentLayoutGeneric
 /sys/fs/cgroup/*/ r,
 /sys/fs/cgroup/*/machine/ rw,
 /sys/fs/cgroup/*/machine/** rw,
+# Non-systemd layout cgroups v2:
+/sys/fs/cgroup/ r,
+/sys/fs/cgroup/machine/ rw,
+/sys/fs/cgroup/machine/** rw,
```

Comment:
It is not.

In cgroupv2 we look for control in everything under: `/sys/fs/cgroup/machine/**`
While in v1, it was `/sys/fs/cgroup/*/machine/**`.

the v1 does not match v2, (it would look work in the case of `/sys/fs/cgroup/machine/machine/**`, which is not the actual paths)

Reply:
When are non-systemd layout cgroups used? Is the comment just confusing and is the intent to say that this is on a system with systemd support (which we require after all) and then we make a cgroup without letting systemd know.

Should this not be described with a delegate unit? https://systemd.io/CGROUP_DELEGATION/